website/allowed-subcategories.txt

```diff
@@ -30,4 +30,5 @@ Secrets Manager
 Security and Compliance Center (SCC)
```

Comment:
@Rakesh-Polepeddi we checked with our DOC team and was asked to use Security and Compliance Center alone and not have Security and Compliance Center (SCC). Let's remove `Posture Management` from this file and also (SCC) just keep  Security and Compliance Center